src/main/resources/org/rust/ide/liveTemplates/output.xml

```diff
@@ -31,6 +31,24 @@
         </context>
     </template>
 
+    <template name="pdv" description="debug println! value" toReformat="true" toShortenFQNames="true"
+              value="println!(&quot;$EXPRESSION_COPY$ = {:?}&quot;, $EXPRESSION$);">
```

Comment:
Wow, it was that easy? 
 
What do you think about merging `pd` and `pdv` templates? We seem to have too may printing related templates now. 

I personally would prefer to have `p` as now and `pd` and `pdd` corresponding to `pdv` and `ppdv`. The reasoning is as follows:

* "Echoing" `pd` and `ppd` perfectly cover printf debuggin scenario.
* `p` covers the lowest common denominator of any other custom printing. 

Providing more specialized templates then `p` seems not very efficient, because there are a lot of ways how you can print something. Inspection to verify the format string ant intentions to switch between `{}`, `{:?}` and `{:#?}` seem a bit more general. 

Thoughts?